bitcoinz.conf

```diff
@@ -14,13 +14,13 @@ addnode=dnsseed.kemperink.org
 addnode=btcz.es
 addnode=btcz.vnminers.com
 server=1
-whitelist=127.0.0.1
+rpcallowip=0.0.0.0/0
```

Comment:
Does binding this to `bitcoinz-insight-patched/32` work instead? If someone runs this without a firewall being open, I can see it leading to potential security concerns.

Reply:
Changed it to `rpcallowip=172.16.0.0/12`